### PR TITLE
Return absolute urls in media

### LINKED ISSUE #2718: Urls returned by GraphQL API should be absolute
### What I'm trying to achieve
Right now default media storage returns relative assets URLs. For instance:
```
/media/__sized__/products/saleor/static/placeholders/t-shirts/6-crop-c0-5__0-5-255x255-70.jpg
```
For API consumers will be much easier to work with relative URLs like:
```
https://example.com/media/__sized__/products/saleor/static/placeholders/t-shirts/6-crop-c0-5__0-5-255x255-70.jpg
```

### Describe a proposed solution
My current solution fixes only product images and looks like:

```
def resolve_thumbnail_url(self, info, *, size=None):
        if not size:
            size = 255
        thumbnail_url = get_thumbnail(self.get_first_image(), size)
        is_absolute_url = urlparse(thumbnail_url).netloc
        if not is_absolute_url:
            return info.context.build_absolute_uri(thumbnail_url) 
        return thumbnail_url
```

But we need a solution which will be working for all media urls.

### PR DESCRIPTION
**Pull request by artursmet**
I want to merge this change because it closes #2718 

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).

### CODE REVIEW

File: saleor/product/templatetags/product_images.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def get_thumbnail_size(size, method):
 
 
 @register.simple_tag()
-def get_thumbnail(instance, size, method='crop'):
+def get_thumbnail(instance, size, method='thumbnail'):
```
Comment (patrys) [inline saleor/product/templatetags/product_images.py]: Could we either drop the method param entirely or make it explicitly required?

File: saleor/graphql/product/types.py [hunk 2]
```diff
@@ -216,10 +216,26 @@ def resolve_products(self, info, **kwargs):
             user=user).filter(product_type=self).distinct()
 
 
+class ImageWithThumbnail(graphene.ObjectType):
```
Comment (patrys) [inline saleor/graphql/product/types.py]: Shouldn't products use this type as well? I think `Image` would be a better name, Shopify has a similar type and it contains the `originalUrl`, a `transformedUrl` (thumbnail), `id` and `altText`.

Comment (maarcingebala) [reply saleor/graphql/product/types.py]: +1 for `Image`

Comment (maarcingebala) [reply saleor/graphql/product/types.py]: Ideally this should be the common interface for all the images exposed by the API, including product images.

**Pull request merged**